pallets/dkg-metadata/src/benchmarking.rs

```diff
@@ -48,8 +48,8 @@ fn mock_signature(pub_key: sr25519::Public, dkg_key: ecdsa::Public) -> (Vec<u8>,
 	(msg, signature.encode())
 }
 
-fn mock_pub_key() -> sr25519::Public {
-	sr25519_generate(AURA, None)
+fn mock_pub_key() -> ecdsa::Public {
+	ecdsa_generate(ECDSA, None)
```

Comment:
It will look like the following once the proper import is given.
```suggestion
	ecdsa_generate(KEY_TYPE, None)
```